eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/VariantMongoAggregationReader.java

```diff
@@ -198,4 +207,38 @@ public void close() throws ItemStreamException {
     @Override
     public void update(ExecutionContext executionContext) throws ItemStreamException {
     }
+
```

Comment:
Section below was moved from [here](https://github.com/EBIvariation/eva-accession/blob/2557d8e6d7f0e4e6d83b8e25ad2814d105c3b3c7/eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/active/AccessionedVariantMongoReader.java#L151) to facilitate shared access to multiple readers.